Essentials/src/config.yml

```diff
@@ -381,6 +381,8 @@ list:
     # Uncomment the line below to simply list all players with no grouping
     #Players: '*'
 
+real-names-on-list: false
```

Comment:
Every config option should have a description. What's clear to you may not be for others.